apps/renderer/src/modules/feed-column/corner-player.tsx

```diff
@@ -259,7 +265,7 @@ const PlayerProgress = () => {
 
           {/* indicator */}
           <Slider.Thumb
-            className="block h-2 w-[3px] rounded-[1px] bg-accent"
+            className="block h-2 w-[3px] cursor-pointer rounded-[1px] bg-accent duration-200 ease-in-out group-hover:size-2"
```

Comment:
```suggestion
            className="block h-2 w-[3px] rounded-[1px] bg-accent duration-200 ease-in-out"
```

prefer native app design system